Apps/APN-UIKit/NotificationServiceExtension/NotificationService.swift

```diff
@@ -7,10 +7,6 @@ class NotificationService: UNNotificationServiceExtension {
     var bestAttemptContent: UNMutableNotificationContent?
 
     override func didReceive(_ request: UNNotificationRequest, withContentHandler contentHandler: @escaping (UNNotificationContent) -> Void) {
-        CustomerIO.initialize(siteId: BuildEnvironment.CustomerIO.siteId, apiKey: BuildEnvironment.CustomerIO.apiKey, region: Region.US) { config in
-            config.autoTrackPushEvents = true
-        }
```

Comment:
I know the CDP SDK changed the NSE public API from using `CustomerIO.initialize` to instead providing the siteid and apikey in the `MessagingPush` module. 

Are you planning on adding this in a future PR?